### ISSUE TITLE
S10e Crashes 3 seconds after booting (bixby, vol up, power -> release way)

### ISSUE THREAD

**Issue by snotrman** [topjohnwu/Magisk#1755]
Can somebody, please help.

- Is there a better place on XDA where to ask this question?

- Where and how can I access any logs, if the system doesn't yet have root access to view system directories nor create a file?

_______________________________

After following your guide at https://topjohnwu.github.io/Magisk/install.html#samsung-system-as-root

0. I patched stock ROM with Magisk Canary for s10s and flashed with Odin 3.13.1. (using bl, cp, csc from the stock image)

1. I booted using (bixby, vol up, power -> release at warning)... There was no Magisk present. 

2. I downloaded Magisk from highonandroid.com. I open the app and everything crashes and becomes unresponsive. (with or without internet connection). (The same crash both with magisk 19.3 and canary). After some minute of holding power button, it boots up again.

3. The second and consecutive times launching into the Magisk system, the system crashes and becomes unresponsive some 3 seconds after launching into UI.

* But if I boot just with power button and download Magisk - when I open it it runs as expected - but of course it shows that it is not installed.

So now I'm thinking.. To patch my stock ROM with the latest 19.4 Magisk, instead of the Canary for Samsung s10s and flash it.

Is that a good idea or will it brick my device?

Is there anything else I can try?

**Comment by snotrman** [comment]
I have a s10e G970F

I possibly made the following mistakes, before I found your guide, following youtube guides.

- The first time I tried to flash patched ROM I used only the AP option with the pached ROM. It froze at "analyzing files".

- then I flashed CP, CSC, BL one by one, in random order.. At one point it stuck at flashing recovery.img, so I unplugged and got into the Samsung emergency recovery screen.

- Then I flashed just unpached AP in one go from stock ROM image an system shrunk from 125GB to 20GB...

- From that state I patched the stock ROM with Magisk 

- I flashed the pached ROM with AP, BL, CSC, CP and I system was back with its 125GB

Maybe some of these mistakes have made Magisk crash the system.

Anything I can do?

Thanks a lot!
